README.md

```diff
@@ -0,0 +1,17 @@
+# browserify-rsa
```

Comment:
it's bad that file systems are insensitive, do you really want to keep lowercase?